src/LowLevelMotionController.cpp

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// LowLevelMotionController
+//
+// Class to implement control of the quads movement
+//
+////////////////////////////////////////////////////////////////////////////
+
+#include <ros/ros.h>
+#include "QuadVelocityController.hpp"
+
+using namespace Iarc7Motion;
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "Low_Level_Motion_Control");
+
+    ROS_INFO("Low_Level_Motion_Control begin");
+
+    ros::NodeHandle nh;
+
+    QuadVelocityController quadController(nh);
+
+    quadController.init();
+
+    while(ros::ok())
+    {
+        quadController.update();
```

Comment:
I have that by specifying a timeout for how long it will wait for a new transform before spitting out ROS_ERROR's.

Also it checks to see that the time between transforms isn't too long.